src/main/java/net/eriagacha/controller/GachaTelemetryController.java

```diff
@@ -40,7 +48,12 @@ public static void InsertTelemetry(String playerName, String rewardObtained) {
                 name.getDate()))
         .flatMap(gr::save);
 
-    saved.subscribe(log::info);
+    try {
+      saved.subscribe();
+    }catch (Exception e){
+      log.error("Erorr de GTC" + e.getMessage());
```

Comment:
Message in spanish and what does GTC mean?

Reply:
Done